netci.groovy

```diff
@@ -103,7 +103,7 @@ set TMP=%TEMP%
 
       def triggerPhraseOnly = configuration == 'release'   
       def triggerPhraseExtra = ""
-                Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto')
+                Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-update3')
```

Comment:
Carry-over from when we had a static machine pool. A proper name would probably be 'auto-update3', but I opted for following the current naming convention.